src/Model/ContextKeys.re

```diff
@@ -6,7 +6,13 @@ let menus =
       bool("listFocus", model => model != None),
       bool("inQuickOpen", model => model != None),
       bool(
-        "quickmenuCursorEnd",
+        "inEditorsPicker",
+        fun
+        | Some({variant: EditorsPicker, _}) => true
+        | _ => false,
+      ),
+      bool(
+        "oni.quickmenuCursorEnd",
```

Comment:
Oni-specific context keys have been namespaced with `oni.` to distinguish he from the "standard" ones from vscode, fore reasons discussed earlier.

Reply:
I missed the reason this is required - is it to avoid conflicts?

Reply:
That, and for both us and end-users to get a better understanding of what's oni-specific and what comes from vscode.